BootloaderCorePkg/Library/SmbiosInitLib/SmbiosTables.h

```diff
@@ -157,6 +157,29 @@ typedef struct {
   EFI_PROCESSOR_FEATURE_FLAGS       FeatureFlags;
 } EFI_PROCESSOR_ID_DATA;
 
+//
+// Add more platform specific strings
+// in the following format
+//
+
+#define TO_BE_FILLED  0
+#define TO_BE_FILLED_STRING  " "        ///< Initial value should not be NULL
+
+///
+/// SMBIOS Table values with special meaning
+///
+#define SMBIOS_TYPE16_USE_EXTENDED_MAX_CAPACITY  0x80000000
+
+#define SMBIOS_TYPE17_UNKNOWN_SIZE               0xFFFF
+#define SMBIOS_TYPE17_USE_EXTENDED_SIZE          0x7FFF
+#define SMBIOS_TYPE17_KB_BIT_MASK                0x8000
+#define SMBIOS_TYPE17_NUMBER_OF_STRINGS               6
```

Comment:
It looks some of these newly added definitions are not used.  do we really need add them?

Reply:
hi @gdong1 , yes it doesn't impact any of the tables, I have discarded them.